src/Commands/Customers/CustomersGroups.php

```diff
@@ -0,0 +1,384 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ */
+
+namespace FOP\Console\Commands\Customers;
+
+use Customer;
+use FOP\Console\Command;
+use Group;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Validate;
+
+/**
+ * This command display common information the latest products.
+ */
+final class CustomersGroups extends Command
+{
+    public const ACTION_DELETE_FROM_GROUP = 1;
+    public const ACTION_REMOVE_CUSTOMERS_TO_FROM_GROUP = 2;
+    public const ACTION_JUST_COPY = 3;
+    public const ACTION_CANCEL = 4;
+
+    public $groupQuestionsKeyPrefix = 'category_';
+    public $actionQuestionsKeyPrefix = 'action_';
+    public $displaySummaryTab = true;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:customer-groups')
+            ->setDescription('Customer groups')
+            ->setHelp('Customer groups')
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $helper = $this->getHelper('question');
+
+        $optionsGroupFrom = $this->getQuestionsOptions('optionsGroupFrom');
+        $optionsGroupTo = $this->getQuestionsOptions('optionsGroupTo');
+
+        if (count($optionsGroupFrom) <= 1) {
+            $output->writeln('<error>You must have at least two groups associated with at least one customer</error>');
```

Comment:
Maybe a notice could be better.
+ a message like 'command aborted' could be good.

Reply:
Done